src/components/badge/badge.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component, h, Prop } from '@stencil/core';
+const ONE_MILION = 1000000;
+const ONE_THOUSAND = 1000;
+const FOUR_DIGIT_LENGTH = 4;
+const FIVE_DIGIT_LENGTH = 5;
+
+@Component({
+    tag: 'limel-badge',
+    styleUrl: 'badge.scss',
+    shadow: true,
+})
+export class Badge {
+    /**
+     * Shows numbers in the badge
+     */
+    @Prop()
+    public badgeCounter: number;
```

Comment:
I think we should rename this. `number`? `label`?

Reply:
Changed to label